data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Stormhold Artifacts [1] Coded Message"
+	name "Go to <planet>"
+	description `Head to <planet> to deal with a "security issue."`
+	landing
+	to offer
+		has "event: stormhold korath issue"
+		not "Hai Reveal [B12] Devil-Hide Battle: offered"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai" "Hai (Unfettered)" "Pirate"
+		not planet Stormhold
+	destination Stormhold
+	on offer
+		conversation
+			`As soon as you land, your communications console beeps urgently. It's a coded message from <planet> that simply reads, "Security issue. Return to <planet> as soon as possible."`
+			`	The digital signature claims to be of diplomatic origin, and from the burning acorn insignia, it is most likely from the Unfettered.`
```

Comment:
I'm not sure about the burning acorn because it's a species from Earth and I can't see why the Unfettered would use it when they might have a better symbol that seems more 'Hai' to me. But Hurleveur says they like it :#

Reply:
The frequent use of acorns is a running gag in Hai missions.

Reply:
It makes as much sense as the Hai naming their ships after Earth creatures: Shield beetles, cicadas, sea scorpions, etc. I just assume that these are linguistic translations, and that the Hai have something that looks like an acorn, and they have species that are sufficiently similar in appearance and/or name to human equivalents that translators just translate them directly. 

It's probably giving biologists fits "No, no, that's not the seed from a *Quercus rubra*, but rather the seed from a Hai tree that we call the *Hai-Home Mercadus Flora Acornus.* Just because it *looks* like an acorn doesn't mean it *is* one."

Reply:
It'd be really amusing to discover that whatever the Hai look-alike to an Acorn has some larger cultural relevance. For instance, perhaps it has some unusual medical properties: It completely suppresses pain and fatigue while inducing a state of high-activity. The sort of thing that you'd give to soldiers or rescue personnel. So eating human acorns has this weird sort of "I look like a tough person eating this thing that usually only people in physically demanding and life-threatening jobs eat, but without the negative of being wired and impossible to sleep for the next day."

Or perhaps it is the inverse: Something with intense calming properties that give Hai a strong "everything is fine, no need to worry" mental state for several hours. Perhaps used in religious ceremonies or something. So Hai enjoy eating human acorns because they are tasty and satisfying the cultural/religious push to consume these things; while avoiding the "mellowed out for several hours" after effect. (Having a significant part of the population routinely self-administering calming substances could be a major contributor to the long-term stability of the Hai)

On the flip side, maybe Hai acorns are just generically commonplace things, much like, say, wheat, rice, or maybe honey for humans. We use many agricultural symbols based on some of these in non-agricultural contexts. And it's the Earth originating acorns that have some decidedly unusual properties that the Hai are *very* fond of. It'd be amusing if that which is a regular nut for humans has strong pharmaceutical properties for the Hai that some like to use recreationaly. Perhaps it is a mild but significant effect, so they don't even recognize it. They just know that they *really* like those acorns from Earth... 

Lots of possibilities of what, exactly, the acorn could represent to the Hai people in general, and to the organizations that use it in specific. Also a lot of amusing and/or meaningful cultural reasons as to why human acorns are so popular.

Reply:
> It makes as much sense as the Hai naming their ships after Earth creatures: Shield beetles, cicadas, sea scorpions, etc. I just assume that these are linguistic translations, and that the Hai have something that looks like an acorn, and they have species that are sufficiently similar in appearance and/or name to human equivalents that translators just translate them directly.
> 
> It's probably giving biologists fits "No, no, that's not the seed from a _Quercus rubra_, but rather the seed from a Hai tree that we call the _Hai-Home Mercadus Flora Acornus._ Just because it _looks_ like an acorn doesn't mean it _is_ one."

Well, that was what I thought the names of the Hai ships were like, too. I only pointed out the acorn because I had thought the Unfettered were around before regular humanity met the Hai, and it just didn't seem right that they'd use a foreign symbol for something that seems to be a nationalistic movement. And yes, it would be much better if these ideas were actually found in game, that would be an interesting conversation for the player.

Reply:
All add a hybrid of what Zitchas and bene-dictator said in my revisions to A11. The revised version will give you a second route through the dialog: read intelligence reports instead of attending the council proceedings, and then get a summary at the end that'll make you wish you were at the proceedings. (This will make more sense when you see the PR.)

I'll add an "intelligence report" about the remarkable similarities between species, and how common biological pressures lead to common evolution of species on different planets. Also, I'll research the physiological effects of various acorn species on squirrels, so I can have plausible technobabble.

Reply:
I don't think a report of the similarities of Human acorns and Hai acorns is very relevant to this Korath mission. I would have thought that it might be more interesting to be it's own mission outside of any campaign; maybe a human scientist wants to study them after Hai Revealed, for example, to create a unique blend that would be commercially successful or to improve medicinal use or whatever. If it's included in this mission, I'd expect people to ignore it so they can get onto the action of destroying Korath faster instead.

Reply:
What? No, not this mission. The super-long Elder Council proceedings that took MasterOfGrey six weeks to write.

Reply:
Yeah, I know, I didn't read it properly.